topic/timeseries/explore/timeseries-queries-and-visualization.ipynb

```diff
@@ -54,7 +54,10 @@
     "## Install required dependencies\n",
     "\n",
     "*Note on Google Colab:* Google Colab currently uses pandas 1.5.3. During the installation\n",
-    "of the dependencies, you will see a compatibility error, which can be ignored."
+    "of the dependencies, you will see a compatibility error, which can be ignored.\n",
+    "\n",
+    "[CrateDB Cloud]: https://console.cratedb.cloud\n",
+    "[Time-Series: Analyzing Weather Data]: ???"
```

Comment:
Not sure what this is supposed to link to. Anyone knows?

Reply:
Let's link it here: https://cratedb.com/docs/cloud/en/latest/tutorials/time-series.html#time-series